common/_usb_rawhid.h

```diff
@@ -0,0 +1,129 @@
+/* Teensyduino Core Library
```

Comment:
So this looks like it was copied from the teensyduino core library.
Why?
Can't we just use the one that was in there?
If not, we need comments that explains why and what was changed.